site/AntDesign.Docs/Demos/Components/Checkbox/doc/index.en-US.md

```diff
@@ -24,14 +24,15 @@ Checkbox
 | Checked            | Specifies whether the checkbox is selected.           | boolean         |false|
 | Disabled            | Disable checkbox           | boolean         |false       |
 | Indeterminate |indeterminate checked state of checkbox       | boolean        | false         |
-| CheckedChange |The callback function that is triggered when the state changes| function(e)|-     |
+| OnChange |The callback function that is triggered when the state changes| function(e)|-     |
 
 Checkbox Group
 
 | Property | Description | Type | Default Value |
 | --- | --- | --- | --- |
 | CheckboxItems | Check box items                             | IList<AntCheckbox>        | -         |
 | Disable | Disable all checkbox                               | boolean        | false         |
+| MixedMode            | Applies only when bot `Options` and `ChildContent` is used. Allows to select which section (`Options` or `ChildContent`) will be rendered first.           | (enum)CheckboxGroupMixedMode         |ChildContentFirst       |
```

Comment:
The parameter is naming `MixMode`.

Reply:
Hmm...It was supposed to be `MixedMode`. Wrong parameter name.